bloom-instance/network/main.tf

```diff
@@ -0,0 +1 @@
+# Not sure if I need this file.
```

Comment:
see comment -- delet this?

Reply:
I would keep the file but move it outside of the module and into the root.  The goal is to have files with meaningful names that represent their content so it's easier to manage resources and develop a cognitive map of where they exist.  One file per resource would clutter things up and make navigating the templates more difficult that way, but you also want to avoid templates that are hundreds of lines long.  Sometimes the happy medium doesn't show itself until you have more things built out, but it's pretty trivial to refactor later if needed.

As an example, consider the network module from the demo.  Inputs and outputs are in their own files to keep them separated.  The VPC is also in it's own file, as well as the IGW which is directly attached to it and only has a dependency on the VPC itself.  Anything that goes into the public subnets or is necessary for them to exist is in its own file.  Likewise, anything needed to fully set up the private subnets is in its own file.  The network setup for the Doorway infrastructure will look pretty similar to this, with the exception that we will be pulling it out of a module, adding one more set of subnets, and might not need an NGW.

Reply:
I think I did within the spirit of this -- moved stuff out of the module and put the vpc logic into vpc.tf. If I interpreted this incorrectly though please feelf ree to reopen!